formatTest/unit_tests/input/infix.re

```diff
@@ -755,3 +755,42 @@ let containingObject = {
     something.contents = "newvalue" [@shouldBeRenderedOnString];
   };
 };
+
+let x = foo |> z;
+
+let x = foo |> f |> g;
+
+let x = foo |> somelongfunctionname "foo" |> anotherlongfunctionname "bar" 1 |> somelongfunction |> bazasdasdad;
```

Comment:
Looks like this should be breaking at some point. It's 113 chars long, yet the print width for test is about 50 chars iirc.